translations/messages+intl-icu.de.yml

```diff
@@ -361,7 +361,7 @@ check.survey.comment.disapprove: "Sind Sie sicher, dass Sie die Veröffentlichun
 check.tag.delete: "Sind Sie sicher, daß Sie das Schlagwort \"{tag}\" löschen möchten?"
 check.topic.delete.tags: "Sind Sie sicher, daß Sie das Thema \"{topic}\" und alle zugeordneten Schlagworte löschen möchten?"
 check.topic.delete: "Sind Sie sicher, daß Sie das Thema und alle Beiträge dazu löschen möchten?"
-check.user.delete: "Sind Sie sicher, dass Sie diese Nutzenden löschen möchten?"
+check.user.delete: "Sind Sie sicher, dass Sie diese(n) Nutzenden löschen möchten?"
```

Comment:
You can use the power of intl-icu and construct something like 
```
check.user.delete: |
    {count
        =1 { Sind Sie sicher, dass Sie diese*n Nutzer*in löschen möchten? }
        other { Sind Sie sicher, dass Sie diese Nutzenden löschen möchten? }
    }
```
You can call that with: Translator.trans('check.user.delete', { count: 1 })

Reply:
True, that's nicer. Thanks for the suggestion. Done in [05edd91](https://github.com/demos-europe/demosplan-core/pull/855/commits/05edd918977857db6833d387e930b05f0cec6cf2)